spacemesh/v1/activation_types.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+package spacemesh.v1;
+option go_package = "github.com/spacemeshos/api/release/go/spacemesh/v1";
+
+import "spacemesh/v1/types.proto";
+
+message GetRequest {
+    bytes id = 1;
+}
+
+message GetResponse {
```

Comment:
is there a benefit of wrapping Activation in GetResponse? i understand that it allows to add more fields (and maybe deprecate field 1) to GetResponse in the future, but what for

Reply:
The only motivation is to be able to change the contents of the response in a backwards-compatible fashion in the future.